misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/AnyAssertionsSpec.kt

```diff
@@ -151,6 +167,36 @@ abstract class AnyAssertionsSpec(
                     expectSubject.isNotSameFun(2)
                 }
             }
+            context("one equals only one of the others") {
+                it("${isNoneOf.name} throws AssertionError") {
+                    expect {
+                        expectSubject.isNoneOfFun(1, arrayOf(2))
+                    }.toThrow<AssertionError> {
+                        message {
+//                            contains(IS_NONE_OF.getDefault())
+                            contains("1")
```

Comment:
```suggestion
                            contains(IS_NONE_OF.getDefault())
                            contains("1")
                            containsNot("2")
```
there is a high chance that this will not pass as `1` and `2` is not enough context (might be in the string nonetheless, e.g. as part of an identity hash).
I suggest you add `listBulletPoint: String,` as argument to the spec (see for instance IterableNoneAssertionsSpec) and then check:
```suggestion
                            contains(IS_NONE_OF.getDefault(), "$listBulletPoint 1)
                            containsNot("$listBulletPoint 2")
```

Reply:
the message doesn't contain `listBulletPoint`, here's the message `message: expected that subject: 1`

Reply:
If that is really all we got as message, then something is wrong. But I am pretty sure there is more, `is none of` should at least be there

Reply:
my bad, the issue is fixed by changing `"${listBulletPoint} 1"` to `"${listBulletPoint}1"`.